Jenkinsfile

```diff
@@ -16,7 +16,6 @@ stage("Build and Publish") {
       sh label: "Build Environment", script: """set -ex
       conda env update -n ${ENV_NAME} -f static/build.yml
       conda activate ${ENV_NAME}
-      pip uninstall -y d2lbook
```

Comment:
This is a workaround for the env cache - in order to integrate the latest features of d2lbook we force to re-install it in this way. Maybe we can keep this for now and iterate over it after we have better ideas of env cache on Actions.

Reply:
I see, makes sense.